WorldBank-Transport/DRIVER
# Add placeholder Thai language

This enables `th` as a valid language code, but does not provide any
translations yet. @tgilcrest is going to add the real translations later this week.

# Testing instructions
- Bring up your VMs
- Launch the front-end development server with `./scripts/grunt.sh web serve`
- Edit `web/app/scripts/config.js` and add the following to the `languages` array: `{"rtl": false, "id": "th", "label": "ภาษาไทย"}`
- Wait for grunt to pick up the change (it may take a bit, this project uses NFS shared folders which aren't super snappy).
- Load the grunt-served frontend at `https://localhost:7002` and log in. 
- Confirm that you can switch the app into Thai using the language switcher menu (since this is a placeholder, all the strings will just be English prefixed with `!`.
- Confirm that you can open the date-picker filters without encountering console errors (which is what would have happened had `date-utils.js` not been edited).

# Notes
- This is hopefully the last time we need to do the `date-utils.js` song-and-dance; that's scheduled for fixup as part of the language refactor.
- This doesn't enable the Thai calender, which is not the same as the Gregorian calendar.
